arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -806,6 +806,14 @@
 				<key>displayName</key>
 				<string>Query related features from non-spatial table</string>
 			</dict>
+			<dict>
```

Comment:
When I tapped on the info button for a row in the sample category table view, it crashes with this error

> Thread 1: "UITableView must be updated via the UITableViewDiffableDataSource APIs when acting as the UITableView's dataSource: please do not call mutation APIs directly on UITableView.

<img width="1122" alt="image" src="https://user-images.githubusercontent.com/9660181/173448637-5e87d109-c40c-448d-8ffb-e473647b9521.png">

I believe it is due to our recent change to #1159 that changed the data source to be diffable. Need to take another look.

Reply:
I see, we'll fix this in another PR.